Main.java

```diff
@@ -9,7 +10,24 @@
    * @param args the command line arguments
    */
   public static void main(String[] args) {
-    // TODO code application logic here
+    //create a scanner for user input
+    Scanner input = new Scanner(System.in);
+
+    //ask the user for the x and y points, declare and initialize the points
+    System.out.println("Please enter in the x coordinate of the first point");
+    int x1 = input.nextInt();
+    System.out.println("Please enter in the y coordinate of the first point");
+    int y1 = input.nextInt();
+    System.out.println("Please enter in the x coordinate of the second point");
+    int x2 = input.nextInt();
+    System.out.println("Please enter in the y coordinate of the second point");
+    int y2 = input.nextInt();
+
+    //calculate, declare and intialize slope
+    int slope = (y2 - y1) / (x2 -x1);
```

Comment:
From the output, we actually know that these should be doubles. The reason we can tell is that the final number had a decimal place in it. Also, I believe one of the sample outputs had a 2.5 as an input. Just be careful! Otherwise, the math and things look good.